plugins/module_utils/botocore.py

```diff
@@ -53,7 +53,12 @@
 from ansible.module_utils.six import binary_type
 from ansible.module_utils.six import text_type
 
+from .exceptions import AnsibleBotocoreError
 from .retries import AWSRetry
+from .version import LooseVersion
+
+MINIMUM_BOTOCORE_VERSION = '1.21.0'
+MINIMUM_BOTO3_VERSION = '1.18.0'
```

Comment:
Can't we pull out these from requirements?

Reply:
As much as I'd like to, unfortunately this is not possible.

requirements.yml is only available on the controller.  For most modules, running in the 'host' context, that file isn't available because it's not passed over to the 'host'.